weblab/entities/migrations/0016_delete_fitting_spec_protocols.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.16 on 2020-11-25 15:02
+
+from django.db import migrations
+
+def delete_protocols(apps, schema_editor):
```

Comment:
I'd call this `delete_old_style_fitting_specs` to make it clearer what it's doing.

Reply:
thanks Jonathan. I have made those changes. Do I need to change/write any unit tests?